src/pranitaPJun20/ArrayFindMissingNum20.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, 
+you need to write a Java program to find that missing number in an array.
+ */
+package pranitaPJun20;
+
+public class ArrayFindMissingNum20 {
+	int sumActual = 0;
```

Comment:
Variable needs to be declared as **local variable.**

Reply:
Yes correct.Made the change.